source/extensions/common/wasm/exports.cc

```diff
@@ -710,6 +717,26 @@ Word wasi_unstable_fd_seek(void*, Word, int64_t, Word, Word) {
 // __wasi_errno_t __wasi_fd_close(__wasi_fd_t fd);
 Word wasi_unstable_fd_close(void*, Word) { throw WasmException("wasi_unstable fd_close"); }
 
+// __wasi_errno_t __wasi_fd_fdstat_get(__wasi_fd_t fd, __wasi_fdstat_t *stat)
+Word wasi_unstable_fd_fdstat_get(void* raw_context, Word fd, Word statOut) {
+  // We will only support this interface on stdout and stderr
+  if ((fd.u64_ != 1) && (fd.u64_ != 2)) {
```

Comment:
Style: you're overusing parentheses here and in tests.

Reply:
Done!